eclcli/vnf/v1/virtual_network_appliance.py

```diff
@@ -369,15 +373,15 @@ def take_action(self, parsed_args):
             data.virtual_network_appliance_plan_id)
         setattr(data, 'virtual_network_appliance_plan', plan.name)
 
-        _set_interface_names_for_display(data)
+        _set_interfaces_for_display(data)
```

Comment:
Fix output format because meta update command does not update interfaces' name now.